src/client/src/consumer.rs

```diff
@@ -390,9 +437,53 @@ impl ConsumerConfig {
     }
 }
 
+pub struct RecordBatch {
```

Comment:
Instead of creating another duplicated struct, can you add to Batch in the data plane

Reply:
Got it. I added new methods `.records()` and `.into_records()` that return iterators over the `DefaultRecord`s inside of the batch.